common/tests/test_simple_page.py

```diff
@@ -60,6 +61,7 @@ def setUp(self):
         }
 
     def test_can_create_simple_page(self):
+        self.home_page = HomePage.objects.first()
```

Comment:
I've noticed on our re-design branch these tests are skipped https://github.com/freedomofpress/pressfreedomtracker.us/commit/bbc2c7258f3197c1a171d5cb1c4f1de6a334bb2b#diff-e1ece5f3301b936378213bf662c40a7b048dcfdd8bebb8501fa857a18e4d6cb3R17

I'm a bit puzzled at the moment how this test was passing when self.home_page wasn't defined :confused: